packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1711,21 +1955,31 @@ export const accountRoutes = (
             ...(!(config as any).isProduction && {
               preVerified: isA.boolean(),
             }),
-          }),
+          })
+          .and('authPWVersion2', 'wrapKb', 'wrapKbVersion2', 'clientSalt'),
         },
         response: {
           schema: isA.object({
             uid: isA.string().regex(HEX_STRING).required(),
             sessionToken: isA.string().regex(HEX_STRING).required(),
             keyFetchToken: isA.string().regex(HEX_STRING).optional(),
+            keyFetchToken2: isA.string().regex(HEX_STRING).optional(),
             authAt: isA.number().integer().description(DESCRIPTION.authAt),
             verificationMethod: validators.verificationMethod
               .optional()
               .description(DESCRIPTION.verificationMethod),
           }),
         },
       },
-      handler: (request: AuthRequest) => accountHandler.accountCreate(request),
+      handler: (request: AuthRequest) => {
+
+        try {
+          return accountHandler.accountCreate(request)
+        }
+        catch (err) {
+          throw new Error('blah', err);
```

Comment:
Maybe include an explanation on why 'blah' also?

Reply:
That shouldn't be there. This debugging cruft, that I missed.